exp/addons/api/v1beta1/clusterresourceset_types.go

```diff
@@ -46,7 +46,7 @@ type ClusterResourceSetSpec struct {
 	Resources []ResourceRef `json:"resources,omitempty"`
 
 	// Strategy is the strategy to be used during applying resources. Defaults to ApplyOnce. This field is immutable.
-	// +kubebuilder:validation:Enum=ApplyOnce
+	// +kubebuilder:validation:Enum=ApplyOnce;Reconcile
```

Comment:
If this strategy is immutable, we need to set it to Reconcile while creating the cluster itself?

Reply:
Yeah!

Reply:
Or if you need to move from a `ApplyOnce` CRS to a `Reconcile` one, just delete the old one (this won't delete the resources in the target clusters) and create the new one with the new strategy

Reply:
Can we add that suggestion to the docs. I can imagine it coming up a lot as the default is `ApplyOnce`

Reply:
Fair enough, done!